kubernetes/aks/system/ori/base/postgresql/service.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ori-postgresql-service
+spec:
+  selector:
+    app: ori-postgresql
+  ports:
+    - port: 5432
+      targetPort: 5432
+  type: ClusterIP
```

Comment:
This will be created when the postgresql database (from cloudnativepg will be deployed)

Reply:
removed